recipes/porechop/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: porechop
+  version: '0.2.3_seqan2.1.1'
+
+source:
+  fn:  Porechop-0.2.3_seqan2.1.1.tar.gz
+  url: https://github.com/jvolkening/Porechop/archive/v0.2.3_seqan2.1.1.tar.gz
```

Comment:
@jvolkening any reason you are using your fork and the the original repo?

Reply:
See below for a general discussion. I'm hoping that Ryan will accept this as a stable branch parallel to master in his repo. When that happens, I will change the source to point to upstream. I'm probably being a little impatient, but I really want to get this into bioconda so that I can polish up the Galaxy wrapper and push it from testing to the main Galaxy toolshed (we have people using it in production and in soon-to-be-published workflows).